lite/examples/gesture_classification/android/app/src/main/java/org/tensorflow/lite/examples/gesture/ImageClassifier.java

```diff
@@ -169,14 +169,14 @@ public void close() {
     List<String> labelList = new ArrayList<String>();
     BufferedReader reader =
         new BufferedReader(new InputStreamReader(activity.getAssets().open(getLabelPath())));
-    String line;
-    line = reader.readLine();
-    reader.close();
-
-    StringTokenizer tokenizer = new StringTokenizer(line, ",");
-    while (tokenizer.hasMoreTokens()) {
-      String token = tokenizer.nextToken();
-      labelList.add(token);
+    while (true){
+        String line = reader.readLine();
+        if (line==null){break;}
```

Comment:
You left out `reader.close();`.

Rather than an extra while loop just use 
```
if (line == null) {
  return labelList;
}
```
Though shouldn't we log an error/warning in that case?

Reply:
Sorry for the `reader.close();`

And here is my labels.txt file
```
UP
DOWN
```
If I use
```
String line;
line = reader.readLine();
reader.close();
```
I got `D/TfLiteCameraDemo: loadLabelList: [UP]`.The `DOWN` label is missing.It only load the first line in labels.txt.

And it causes the error :

`java.lang.IllegalArgumentException: Cannot copy between a TensorFlowLite tensor with shape [1, 2] and a Java object with shape [1, 1].`

Reply:
Right, it looks like the current format expects a single line with comma-separated labels. Can you simple make your labels.txt file "UP,DOWN"?

Reply:
Sure,thanks